typescript/cli/src/commands/warp.ts

```diff
@@ -275,24 +277,55 @@ const send: CommandModuleWithWriteContext<
     warp,
     amount,
     recipient,
+    roundTrip,
   }) => {
     const warpCoreConfig = await getWarpCoreConfigOrExit({
       symbol,
       warp,
       context,
     });
 
+    let chains: ChainName[] = warpCoreConfig.tokens.map((t) => t.chainName);
+    if (roundTrip) {
+      // Appends the reverse of the array, excluding the 1st (e.g. [1,2,3] becomes [1,2,3,2,1])
+      const reversed = [...chains].reverse().slice(1, chains.length + 1); // We make a copy because .reverse() is mutating
+      chains.push(...reversed);
+    } else {
+      // Assume we want to use use `--origin` and `--destination` params, prompt as needed.
+      const chainMetadata = objFilter(
+        context.chainMetadata,
+        (key, _metadata): _metadata is any => chains.includes(key),
```

Comment:
Maybe a small optimization but I think it would be better to create a set and use the `.has` method to be a bit more efficient here